lib/3scale/backend/storage_async/methods.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  module Backend
+    module StorageAsync
+      module Methods
+        # Now we are going to define the methods to run redis commands
+        # following the interface of the redis-rb lib.
+        #
+        # These are the different cases:
+        #   1) Methods that can be called directly. For example SET:
+        #      @redis_async.call('SET', some_key)
```

Comment:
```suggestion
        #      call('SET', some_key)
```
Not sure if that's correct, but assuming that there is no mention of `@redis_async` in this file, and the calls are "delegated" to `@redis_async` anyway...

Reply:
You're right, I copied that comment from the old file and didn't check it.